### PR TITLE
Update BerchCoin Metadata

### PR DESCRIPTION
**Pull request by bapshago**
# Pull Request Template

## Description

Update BerchCoin Metadata to include logo

## Type of change

- [ x] Metadata related change
- [ ] Other

## Checklist:

- [ ] For metadata related changes, this PR code passes the Github Actions metadata validation

## Metadata PRs

Please note it may take up to 4 hours for merged changes to take effect on the metadata server.

### CODE REVIEW

File: mappings/c4b65048fd8319a959690d59f66721d638865050cb90765b626309204265726368436f696e.json [hunk 1]
```diff
@@ -1,30 +1,52 @@
...
+            {
+                "signature": "b747b7366c61bc8148834224a08fc5183c256fbdb13432570bebd9dca2b7e3f3c265c594fb7cab369d0e91590c3082ddc552f8d897f7f67d7d5c2511e738b90a",
+                "publicKey": "c3d39acf2d48273bb13e88ff2aec16db8b013f1acea4ca1577dd8ddfc38bcd60"
+            },
+            {
+                "signature": "a214ea41c644ee5e2d5b709e0a9b86e8e4f7a82bfce097774d3a059e48901e39a2e68af35c3b3990c0d65e8c6ca880f46c56fdb1bc9cc2f112e9307acd418e01",
+                "publicKey": "7e71d327449e0af3c03952168017938e7a4605feaf62da81d33b0baad0fbfcc4"
+            },
+            {
+                "signature": "b747b7366c61bc8148834224a08fc5183c256fbdb13432570bebd9dca2b7e3f3c265c594fb7cab369d0e91590c3082ddc552f8d897f7f67d7d5c2511e738b90a",
```
Comment (crptmppt) [inline mappings/c4b65048fd8319a959690d59f66721d638865050cb90765b626309204265726368436f696e.json:31]: duplicate signatures in there - can you remove the extra ones?

Comment (bapshago) [reply mappings/c4b65048fd8319a959690d59f66721d638865050cb90765b626309204265726368436f696e.json]: Duplicate sig removed

**Review by bapshago — COMMENTED** [review]
Updated

File: mappings/c4b65048fd8319a959690d59f66721d638865050cb90765b626309204265726368436f696e.json (hunk 1 above)
Comment (crptmppt) [reply mappings/c4b65048fd8319a959690d59f66721d638865050cb90765b626309204265726368436f696e.json]: it still shows up - holding on merging due to unexpected signatures / behavior of the CI checks

**Pull request closed without merging**